pkg/bot.go

```diff
@@ -464,9 +475,14 @@ func (b *bot) hasPreviewSiteCapability(email, org string) (bool, error) {
 }
 
 func (b *bot) previewSite(args ResponseRequest) string {
+	if args.Email == "" {
+		CreatePreviewSiteCounter.WithLabelValues(b.annotation, metricLabelPreviewDeniedMissingEmail).Inc()
+		return previewDeniedMissingEmail
+	}
 	list, err := b.sisLister.List(labels.Everything())
 	if err != nil {
 		klog.Errorf("failed listing SiteImageSource: %v", err)
+		CreatePreviewSiteCounter.WithLabelValues(b.annotation, metricLabelPreviewFailed).Inc()
```

Comment:
Not sure it matters, however it may be easier to alert on separated.  Also prometheus advise against labels generally, however I think that advice is more around dynamic things you cannot measure like the name of pods.

![image](https://user-images.githubusercontent.com/1883554/104052192-91f36300-51a6-11eb-947b-991c8a6b94bc.png)
![image](https://user-images.githubusercontent.com/1883554/104052209-991a7100-51a6-11eb-8bcf-bce113f078b9.png)

However also an alert doing the same:
![image](https://user-images.githubusercontent.com/1883554/104052243-a6376000-51a6-11eb-9590-17277f11682a.png)

Reply:
> any reason to not have a separate timeseries for each status?

no strong reason really, I loosely modelled this after `htttp_requests_total`. That metric also has `status` in one of its labels and my thinking was "yeah, that sound like similar usage". I think both will behave similarly, in both cases, there is going to be own time series, likely going to consume a similar amount of memory. Some queries are going to be harder, with a label, some easier.

> Also prometheus advise against labels generally, however I think that advice is more around dynamic things you cannot measure like the name of pods.

https://prometheus.io/docs/practices/naming/#labels, that is how I am interpreting their advice as well. This case, when I have a small bounded number of labels, I think is in correlation with general usage and practices.

Weakly leaning towards keeping it as a label but happy to revert to own TS.